Delete index not working

I can't seem to get this code to work:

```
collection.indexesManager.delete("indexName")
```

I get:

```
reactivemongo.core.commands.DefaultCommandError: BSONCommandError['exception while deserializing this command's result!'] with original doc {
  nIndexesWas: BSONDouble(2.0),
  ok: BSONDouble(1.0)
}
    at reactivemongo.core.commands.CommandError$.apply(commands.scala:136) ~[reactivemongo_2.10-0.9.jar:0.9]
    at reactivemongo.core.commands.BSONCommandResultMaker$class.apply(commands.scala:82) ~[reactivemongo_2.10-0.9.jar:0.9]
    at reactivemongo.core.commands.DeleteIndex$ResultMaker$.apply(commands.scala:575) ~[reactivemongo_2.10-0.9.jar:0.9]
    at reactivemongo.api.DB$$anonfun$command$1.apply(database.scala:75) ~[reactivemongo_2.10-0.9.jar:0.9]
    at reactivemongo.api.DB$$anonfun$command$1.apply(database.scala:75) ~[reactivemongo_2.10-0.9.jar:0.9]
    at reactivemongo.utils.EitherMappableFuture$$anonfun$mapEither$1.apply(utils.scala:57) ~[reactivemongo_2.10-0.9.jar:0.9]
Caused by: java.util.NoSuchElementException: None.get
    at scala.None$.get(Option.scala:313) ~[scala-library.jar:na]
    at scala.None$.get(Option.scala:311) ~[scala-library.jar:na]
    at reactivemongo.core.commands.DeleteIndex$ResultMaker$$anonfun$apply$9.apply$mcI$sp(commands.scala:577) ~[reactivemongo_2.10-0.9.jar:0.9]
    at reactivemongo.core.commands.DeleteIndex$ResultMaker$$anonfun$apply$9.apply(commands.scala:577) ~[reactivemongo_2.10-0.9.jar:0.9]
    at reactivemongo.core.commands.DeleteIndex$ResultMaker$$anonfun$apply$9.apply(commands.scala:577) ~[reactivemongo_2.10-0.9.jar:0.9]
    at scala.Option.toLeft(Option.scala:291) ~[scala-library.jar:na]
```

The code at core/commands/commands.scala:577 seems to be looking for a key **nIndexWas**, but my server is returning **nIndexesWas**.

--- a/driver/src/main/scala/core/commands/commands.scala
+++ b/driver/src/main/scala/core/commands/commands.scala
@@ -573,6 +573,5 @@ case class DeleteIndex(
 
   object ResultMaker extends BSONCommandResultMaker[Int] {
     def apply(document: BSONDocument) =
-      CommandError.checkOk(document, Some("deleteIndexes")).toLeft(document.getAs[BSONDouble]("nIndexWas").map(_.value.toInt).get)
-  }
+      CommandError.checkOk(document, Some("deleteIndexes")).toLeft(document.getAs[BSONDouble]("nIndexesWas").map(_.value.toInt).get) }
 }

